File: mylibrary/src/main/java/com/incetro/mylibrary/data/database/MeasureDao.java
package com.incetro.mylibrary.data.database;

import androidx.room.Dao;
import androidx.room.Insert;
import androidx.room.Query;

import com.incetro.mylibrary.model.Measure;

import java.util.List;

@Dao
public interface MeasureDao {
 
   @Query("SELECT * FROM measure")
   List<Measure> getAll();
 
   @Query("SELECT * FROM measure WHERE id = :id")
   Measure getById(long id);

   @Insert
   void insert(Measure employee);
 
}

File: mylibrary/src/main/java/com/incetro/mylibrary/data/database/MeasureDao.kt
package com.incetro.mylibrary.data.database

import androidx.room.Dao
import androidx.room.Insert
import androidx.room.OnConflictStrategy
import androidx.room.Query
import com.incetro.mylibrary.model.Measure

@Dao
interface MeasureDao {

    @Query("SELECT * FROM measure")
    suspend fun getAll(): List<Measure>

    @Insert
    suspend fun insert(employee: Measure)

    @Insert(onConflict = OnConflictStrategy.REPLACE)
    suspend fun insertAll(users: List<Measure>)
}